README.md

```diff
@@ -247,13 +247,16 @@ Cookie jars are provided by the [tough-cookie](https://www.npmjs.com/package/tou
 
 ### Intervening before parsing
 
-jsdom allows you to intervene in the creation of a jsdom very early: after the `Window` and `Document` objects are created, but before any HTML is parsed to populate the document with nodes:
+jsdom allows you to intervene in the creation of a jsdom very early: after the `Window` and `Document` objects are created, but before any HTML is parsed to populate the document with nodes. The callback is also called with new `Window` objects created for each frame:
 
 ```js
 const dom = new JSDOM(`<p>Hello</p>`, {
-  beforeParse(window) {
+  beforeParse(window, main) {
```

Comment:
Can we make the API `{ isMain }` instead of a boolean? See https://ariya.io/2011/08/hall-of-api-shame-boolean-trap